src/components/uploadAreaDnD.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+// Element for handling file staging before uploading using drag and drop or file select
+function UploadAreaDnD({
+  dragging, files, fileAdded, dragEnter, dragLeave, dragDrop, removeFile,
+}) {
+  const uploadArea = (
+    <div className={`uploadAreaDnD ${dragging ? 'dragging' : ''}`} onDragOver={(e) => { e.preventDefault(); e.stopPropagation(); }} onDragEnter={dragEnter} onDragLeave={dragLeave} onDrop={(e) => { e.preventDefault(); e.stopPropagation(); dragDrop(e); }}>
+      <div className="row fileArea">
+        <FileArea files={files} removeFile={removeFile} />
+      </div>
+      <div className="selectArea">
+        <label htmlFor="fileUpload" id="fileUploadLabel" className="btn btn-primary">
+          Select Files
+          <input type="file" name="fileUpload" id="fileUpload" onChange={fileAdded} multiple />
+        </label>
+      </div>
+    </div>
+  );
+  return uploadArea;
+}
+
+// Div populated with staged files, instructions displayed if no files
+function FileArea({ files = [], removeFile }) {
+  if (!files.length) {
+    return (
+      <div className="centered noFiles">
+        <br />
+        <br />
+        <br />
+        <br />
```

Comment:
Using CSS to control vertical spacing is more preferable than `<br />` tags.

Reply:
implemented! Will continue to monitor for this and change in future PRs as well.